llvm/tools/llvm-cas/Options.td

```diff
@@ -0,0 +1,63 @@
+include "llvm/Option/OptParser.td"
+
+class F<string name> : Flag<["--", "-"], name>;
+
+def grp_action : OptionGroup<"Actions">, HelpText<"llvm-cas actions">;
```

Comment:
Have you considered using subcommands for actions? The subcommand support was introduced in https://github.com/llvm/llvm-project/pull/155026.

Reply:
Is it possible to migrate them over the course of a release or two?

Reply:
We should be able to support both syntaxes to avoid breaking existing uses and mark the flag-based versions as deprecated so new users are encouraged to use the subcommand syntax. I'm fine doing it in a follow up change if you prefer.

Reply:
@petrhosek I think subcommand is not usable with positional arguments. That is a deal breaker for now to use subcommand parsing. 

It would also be nice if a flag can be alias to a subcommand but that is not strictly needed.

Reply:
file issue: https://github.com/llvm/llvm-project/issues/166830